packages/valory/skills/mech_interact_abci/behaviours/request.py

```diff
@@ -294,7 +305,13 @@ def async_act(self) -> Generator:
         with self.context.benchmark_tool.measure(self.behaviour_id).local():
             if not self._mech_requests:
                 payload = MechRequestPayload(
-                    self.context.agent_address, None, None, GNOSIS_CHAIN_ID, None, None
+                    self.context.agent_address,
+                    self.matching_round.auto_round_id(),
+                    None,
+                    None,
+                    GNOSIS_CHAIN_ID,
+                    None,
+                    None,
```

Comment:
This won't trigger the `none_event`.

Reply:
This was the case in the previous state of the repository (GNOSIS_CHAIN_ID was also present). Please open an issue to fix it in another PR.

Reply:
Additionally, no NONE event is specified in the FSM app:
https://github.com/valory-xyz/IEKit/blob/main/packages/valory/skills/mech_interact_abci/rounds.py#L51-L56

Reply:
Added issue https://github.com/valory-xyz/IEKit/issues/155. This has to be addressed on a different PR, as this one is already quite complex.